lib/pages/player/player_item_panel.dart

```diff
@@ -507,7 +507,7 @@ class _PlayerItemPanelState extends State<PlayerItemPanel> {
           (Utils.isDesktop() || !videoPageController.isFullscreen)
               ? Container()
               : Positioned(
-                  right: 0,
+                  right: MediaQuery.of(context).padding.right,
```

Comment:
```suggestion
                  right: 0,
```

参考下面自定义底部组件，在 `SlideTransition` 内用 `SafeArea`

```dart
child: SlideTransition(
  position: leftOffsetAnimation,
    child: SafeArea(
      top: false,
      bottom: videoPageController.isFullscreen,
      left: false,
      right: videoPageController.isFullscreen,
      child: Column(
```